src/app/public/test-utility/test-utility-dom-event-options.ts

```diff
@@ -2,4 +2,5 @@ export interface SkyAppTestUtilityDomEventOptions {
   bubbles?: boolean;
   cancelable?: boolean;
   keyboardEventInit?: KeyboardEventInit;
+  additionalEventProperties?: object;
```

Comment:
Think we could borrow from the `CustomEvent` API and change this to `customEventInit?: any`?
https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent

Which event properties do you need to support? Are they associated with a specific `Event` API?

Reply:
Ah yeah, that's a better name. I'll switch it. I don't really like that "detail" property, it seems like an unnecessary extra level.

The property I specifically need is associated with the focusin event api, but it seems far more useful to allow custom property input so that we need not implement and maintain all existing events.